### ISSUE TITLE
Kidney transplant (ICD-10)

### ISSUE THREAD

**Issue by viyaasan** [opensafely/codelist-development#208]
[ICD-10 kidney transplant codelist](https://www.opencodelists.org/codelist/user/viyaasan/kidney-transplant/1a6e58c9/)

**Comment by ss808** [comment]
Z940 Kidney transplant status?

**Comment by viyaasan** [comment]
I don't think we should use this one as it might be to indicate someone is unfit for transplant or not transplanted?
